blackbox/docs/administration/hba.txt

```diff
@@ -0,0 +1,118 @@
+.. _administration_hba:
+
+=========================
+Host Based Authentication
+=========================
+
+.. NOTE::
+
+   Host based Authentication is an Enterprise Edition feature.
+   This feature only concerns connections over :ref:`postgres_wire_protocol`.
+
+This topic explains how to configure client connection and authentication to CrateDB.
```

Comment:
"configure client connection and authentication to CrateDB" -> "configure CrateDB client connection and authentication"